tests/lib/tools/suite/tests.backup/task.yaml

```diff
@@ -0,0 +1,63 @@
+summary: tests for the tests.backup tool
+
+prepare: |
+    mv "${PWD}.tar" "${PWD}.tar.back"
```

Comment:
Can you please add a comment that explains how ${PWD}.tar exists at this stage? One reading the prepare logic might be surprised that this doesn't fail. It assumes some knowledge of how the suite itself is prepared.

Reply:
done